DEVELOPERS.txt

```diff
@@ -0,0 +1,85 @@
+------------- How is image/icon/cover passed from the OPML item to a track -------------
+
+When a $url from an OPML item is added to current playlist, a $track object is created 
+and its image is obtained by calling handlerForUrl($url)->getMetadataFor($url). 
+If this protocol handler is HTTP, it needs to find the image just by using this $url, 
+there is no other information. 
+
+XMLBrowser(s), when adding the url to the playlist, call setRemoteMetadata which caches
+the OPML items's image link as "remote_image_$url". Inside setRemoteMetadata, the $url's
+protocol handler will be called with a method cacheImage(class, url, image, prefix). 
```

Comment:
This documentation seems to document things which aren't implemented yet: I can't find `cacheImage()` yet. I guess that's something you implemented in a different PR?

Reply:
Done